start.sh

```diff
@@ -3,23 +3,38 @@
 # Use 'start.sh -remote' if you want to access the server from another machine (or a VM)
 SERVER="localhost:8082"
 
+if [ $# -gt 0 ] && [ $1 = '-help' ]
+then
+    echo "Usage: start.sh (PHP web server will be listening to localhost:8082)"
+    echo "Usage: start.sh -remote (PHP web server will be listening to 0.0.0.0:8082 and accessible from the outside)"
+    echo "Usage: start.sh -tests (Run all the unit and fonctional tests and check PHP syntax with PHP-cs-fixer)"
+    echo "Additional parameters will be ignored."
+    exit 1
+elif [ $1 != '-remote' ] && [ $1 != '-tests' ] && [ $1 != '-tests-server' ]
```

Comment:
Never mind, I didn't realize it was an option later on, somehow my brain thought it was the concatenation of two options...